jina/parsers/hubble/new.py

```diff
@@ -0,0 +1,70 @@
+"""Argparser module for hub new"""
+
+import argparse
+
+from ..helper import add_arg_group
+
+
+def mixin_hub_new_parser(parser):
+    """Add the arguments for hub new to the parser
+    :param parser: the parser configure
+    """
+    gp = add_arg_group(parser, title='New')
+    gp.add_argument(
+        '-n',
+        '--name',
+        help='Setup name of the executor',
+        action='store',
+        type=str,
+        default=None,
+    )
+
+    gp.add_argument(
+        '-p',
+        '--path',
+        help='Path to store an executor',
+        action='store',
+        type=str,
+        default=None,
+    )
+
+    gp.add_argument(
+        '-ac',
+        '--advance_configuration',

```

Comment:
dash over underscore
```suggestion
        '--advance-configuration',

```